web/impact/impact/graphql/types/base_user_profile_type.py

```diff
@@ -31,3 +38,24 @@ def resolve_program_roles(self, info, **kwargs):
             startup_roles_of_interest += [StartupRole.ENTRANT]
         return get_user_program_and_startup_roles(
             self.user, user_roles_of_interest, startup_roles_of_interest)
+
+    def resolve_office_hour_locations(self, info, **kwargs):
+        family_ids = Clearance.objects.clearances_for_user(
+            self.user
+        ).values_list("program_family", flat=True)
+        desired_user_roles = [
+            UserRole.MENTOR, UserRole.FINALIST, UserRole.AIR]
+        program_family_ids = self.user.programrolegrant_set.filter(
+            program_role__program__program_status="active",
+            program_role__user_role__name__in=desired_user_roles
+            ).values_list(
+                "program_role__program__program_family", flat=True
+                ).distinct()
+        ids = list(set(chain(family_ids, program_family_ids)))
+        remote = Location.objects.filter(name__icontains='remote').first()
```

Comment:
This implementation is prone to errors. Other locations ie `<Location Jerusalem | Israel Remote>` have remote in their name.